client/src/pages/about/page.tsx

```diff
@@ -1,10 +1,19 @@
 import * as React from 'react';
+import MainLayout from '../../components/MainLayout';
+import CardContainer from '../../components/CardContainer';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
do we need this? we already have it on index.tsx

Reply:
yea you are right the bootstrap import is not needed, i'll remove it